enhancements/ingress/aws-lb-eip-selection.md

```diff
@@ -0,0 +1,474 @@
+---
+title: aws-lb-eip-selection
+authors:
+- "@miheer"
+reviewers:
+- "@gcs278"
+- "@alebedev87"
+- "@Miciah"
+- "@mtulio"
+- "@candita"
+- "@frobware"
+- "@rfredette"
+approvers:
+- "@patrickdillon"
+- "@mtulio"
+- "@r4f4"
+- "@sadasu"
+api-approvers:
+- None
+creation-date: 2024-09-26
+last-updated: 2024-10-28
+tracking-link:
+  - https://issues.redhat.com/browse/CORS-3687
+see-also:
+  - ""
+replaces:
+  - ""
+superseded-by:
+---
+
+# Installer LoadBalancer EIP Selection for AWS
+
+## Summary
+
+This enhancement extends the OpenShift Installer's install-config, enabling cluster admins to
+configure EIPs for AWS NLB load balancer created for their `default` NLB IngressController at install time.
+
+## Definitions and Terminology
+
+### The `default` IngressController
+
+This proposal will refer to the IngressController that gets created automatically during installation and handles
+the platform routes (console, auth, canary, etc.) as the `default` IngressController.
```

Comment:
Can you mention whether this feature makes the installation a UPI (user-provisioned) rather than IPI installation?

For example, will the configuration settings be listed in the parameter docs:
https://docs.openshift.com/container-platform/4.17/installing/installing_aws/installation-config-parameters-aws.html#installation-config-parameters-aws

Reply:
i made some modification :
This proposal will refer to the IngressController that gets created automatically during user provisioned installation and handles
the platform routes (console, auth, canary, etc.) as the `default` IngressController.